### PR TITLE
Fix nomination-list in assignment poll track usage

### LINKED ISSUE #4094: Fix display of most current Nomination list (election polls)
**Describe the bug**
The most current election poll displays a too big list (double the items)
Screenshot_20240909_120321

**How to Reproduce**
1. create a new election
2. put a candidate onto the candidate list 
3. create a poll with voting method: Yes/No/Abstain per list 
4. Nomination list has empty items
5. create a new poll with voting method: Yes/No/Abstain per list
6. The newest polls nomination list has the same problems the old one(s) are correct again

**Expected behaviour:**
The nomination list should only display the candidates

### PR DESCRIPTION
**Pull request by reiterl**
Resolve #4094

### CODE REVIEW

File: client/src/app/site/pages/meetings/pages/assignments/modules/assignment-poll/components/assignment-poll-meta-info/assignment-poll-meta-info.component.html [hunk 1]
```diff
@@ -28,14 +28,14 @@
                         {{ 'Candidates' | translate }}):
                         @if (enumerateCandidates) {
                             <ol>
-                                @for (title of poll.options[0]?.contentTitlesAsSortedArray; track title) {
+                                @for (title of poll.options[0]?.contentTitlesAsSortedArray; track title.title) {
```
Comment (bastianjoel) [inline client/src/app/site/pages/meetings/pages/assignments/modules/assignment-poll/components/assignment-poll-meta-info/assignment-poll-meta-info.component.html:31]: AFAIK title is not guaranteed to be unique in that case. In doubt you can always use `track $index`

File: client/src/app/site/pages/meetings/pages/assignments/modules/assignment-poll/components/assignment-poll-meta-info/assignment-poll-meta-info.component.html [hunk 2]
```diff
@@ -28,14 +28,14 @@
...
                                     <li>{{ title.title }}{{ title.subtitle ? ' (' + title.subtitle + ')' : '' }}</li>
                                 }
                             </ol>
                         }
                         @if (!enumerateCandidates) {
                             <ul>
-                                @for (title of poll.options[0]?.contentTitlesAsSortedArray; track title) {
+                                @for (title of poll.options[0]?.contentTitlesAsSortedArray; track title.title) {
```
Comment (bastianjoel) [inline client/src/app/site/pages/meetings/pages/assignments/modules/assignment-poll/components/assignment-poll-meta-info/assignment-poll-meta-info.component.html:38]: Same as above.

**Review by bastianjoel — CHANGES_REQUESTED** [review]
Could you actually reproduce the issue and verify that it was solved with this? 

From the angular docs I only read that using bad values could impact performance and the placement of objects in these track parameters was done by the automatic migration. Normally these should not break anything without at least a note in the migration guide.

**Comment by reiterl** [comment]
Yes, in my test the problems were fixed thru the track improvement.

File: client/src/app/site/pages/meetings/pages/assignments/modules/assignment-poll/components/assignment-poll-meta-info/assignment-poll-meta-info.component.html (hunk 1 above)
Comment (reiterl) [reply client/src/app/site/pages/meetings/pages/assignments/modules/assignment-poll/components/assignment-poll-meta-info/assignment-poll-meta-info.component.html]: use now $index

File: client/src/app/site/pages/meetings/pages/assignments/modules/assignment-poll/components/assignment-poll-meta-info/assignment-poll-meta-info.component.html (hunk 2 above)
Comment (reiterl) [reply client/src/app/site/pages/meetings/pages/assignments/modules/assignment-poll/components/assignment-poll-meta-info/assignment-poll-meta-info.component.html]: Use now $index.

**Comment by reiterl** [comment]
I could reproduce the problem with following the steps description. Also saw a lot of errors about the wrong 'track' behavior. After fixing the track problems, the bug was also gone.

**Review by bastianjoel — APPROVED** [review]

**Review by Elblinator — APPROVED** [review]

**Pull request merged**